main.py

```diff
@@ -33,15 +33,16 @@ def main(sentinel, repos, token, since):
     total_reviews = []
     for pr in to_check:
         for comment in pr.get_issue_comments():
-            if sentinel in comment.body:
-                total_reviews.append(comment.user.name)
+            for sentinel in sentinels:
+                if sentinel in comment.body:
+                    total_reviews.append(comment.user.name)
 
     counter = collections.Counter(total_reviews)
     print '\n'.join(['%s: %s' % (x, y) for x,y in counter.most_common()])
 
 if __name__ == '__main__':
     parser = argparse.ArgumentParser(description="Count reviews on github")
-    parser.add_argument('--sentinel', dest='sentinel', default=':+1:', help='String to look for to count something as "reviewed"')
+    parser.add_argument('--sentinels', dest='sentinels', nargs="*", default='[":+1:", ":metal:"]', help='One or more strings to look for to count something as "reviewed"')
```

Comment:
I'm assuming the behavior is set by argparse's nargs flag, and that indicates that this is a space-delimited list. See: https://docs.python.org/3/library/argparse.html#nargs